packages/@aws-cdk/aws-lambda/lib/lambda-version.ts

```diff
@@ -28,6 +41,11 @@ export interface VersionProps {
   readonly lambda: IFunction;
 }
 
+export interface VersionAttributes {
+  readonly version: string;
```

Comment:
Docstrings

Reply:
added. we'll do another docstrings pass later.